src/cmd/config.go

```diff
@@ -41,6 +43,7 @@ func GetConfigFromEnvironment() *Config {
 				",",
 			),
 		},
+		ProxySupport: envBool("PROXY_PROTOCOL", false),
```

Comment:
I think this needs more consistent naming.  The environment variable `PROXY_PROTOCOL` is good, but `ProxySupport` is quite ambiguous in the context of something that is already a HTTP proxy.

I'm having trouble finding definitive information as to whether "proxy protocol" is a HAProxy feature, or an internet [psuedo-]standard.  It seems at least the CloudFlare and some others support it, so with that in mind it's probably best to refer to it as "proxy protocol" everywhere, and not "haproxy".

Reply:
Changed to `ProxyProtocol`